samza-kafka/src/main/java/org/apache/samza/system/kafka/KafkaSystemConsumer.java

```diff
@@ -66,7 +66,7 @@
 
   // This proxy contains a separate thread, which reads kafka messages (with consumer.poll()) and populates
   // BlockingEnvelopMap's buffers.
-  final private KafkaConsumerProxy proxy;
+  final private KafkaConsumerProxy<K, V> proxy;
```

Comment:
nit: s/final private/private final.

Reply:
Oops, I should have taken the opportunity to do that since I was changing this line anyways, but the PR already got merged. It was already like that anyways, and it can be done in some other PR.